docs/apko_file.md

```diff
@@ -196,3 +196,17 @@ The `paths` element contains the following children:
  - `permissions`: file permissions to set. Permissions should be specified in octal e.g. 0o755 (see `man chmod` for details).
  - `source`: used in `hardlink` and `symlink`, this represents the path to link to.
  
+
+### Includes
+
+`include` defines a path to a configuration file which should be used as the base configuration,
+the configuration data is layered on top of this base configuration.  By default, there is no
+base configuration used.
+
+The path can be either a local file, or a file in a remote git repository, in the same style as
```

Comment:
Given the TODO below I think this could be a little clearer about what the expected format is. It'll work for GitHub and Gitlab, but it wouldn't necessarily work for my own custom Git server.

Reply:
How does golang do it, do you happen to know?

Reply:
I don't actually, they might iteratively try to `git clone example.com/foo/bar/baz` then `example.com/foo/bar` then `.../foo` and `/` until it works? I haven't really tried it.